app/scripts/init_sentiment_db.py

```
@@ -0,0 +1,73 @@
+        print("\n3. Testing database connection...")
+        server_info = await client.server_info()
+        print(f"  ✅ Connected to MongoDB {server_info['version']}")
+
+        print("\n" + "=" * 50)
+        print("✅ Database initialization completed successfully!")
+
+        # Close connection
+        client.close()
+
+    except Exception as e:
+        logger.error(f"Error initializing database: {str(e)}")
```

Ruff (0.13.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)